modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -213,21 +214,36 @@ public void execute() throws MojoExecutionException {
             }
         }
 
-        File ignoreClassesFile = new File(
-            ModernizerAnnotationOutput.getOutputDir(outputDirectory),
-            ModernizerAnnotationOutput.IGNORE_CLASSES_FILE_NAME);
-        if (ignoreClassesFile.exists()) {
-            Collection<String> ignoreClasses =
-                readExclusionsFile(ignoreClassesFile.toString());
-            for (String ignoreClass : ignoreClasses) {
-                allIgnoreFullClassNamePatterns.add(
-                    Pattern.compile(ignoreClass));
-            }
+        String ignoreClassesFileName =
```

Comment:
Can we just use `ModernizerAnnotationOutput.IGNORE_CLASSES_FILE_NAME` instead of this local variable, or does that ruin line-length checkstyle?

Reply:
No. I was using it a couple of times. But I don't think I need to use it more than once. Will refactor this.